src/collectors/elasticsearch/elasticsearch.py

```diff
@@ -209,11 +212,35 @@ def collect_instance_index_stats(self, host, port, metrics):
             indices = result['indices']
         else:
             return
+        
+        if 'aliases' in self.config['stats']:
+          # remove indices not aliased
+          # aggregate values from aliased indexes
+          aliases={}
```

Comment:
declare `aliases` before the if block, then you don't need the `else` or the initial declaration in the `if